docs/dev/dev_setup.rst

```diff
@@ -79,40 +87,24 @@ listed in the project's ``pyproject.toml`` file.  The conda lockfile is updated
 automatically by a GitHub Action workflow that runs once a week, or any time
 ``pyproject.toml`` is changed.
 
-Use ``conda-lock`` to create a new ``pudl-dev`` environment and activate it:
-
-.. code-block:: console
-
-    $ mamba activate base
-    $ conda-lock install --name pudl-dev --mamba --dev environments/conda-lock.yml
-    $ mamba deactivate
-    $ mamba activate pudl-dev
```

Comment:
I removed these details entirely, and defaulted to the instructions which use `make`.  If folks want to see the detailed commands, they are listed in the `Makefile`

Reply:
i think that's great